system/ee/ExpressionEngine/Controller/Addons/Addons.php

```diff
@@ -31,6 +32,7 @@ class Addons extends CP_Controller
     public function __construct()
     {
         parent::__construct();
+        ee()->cp->add_js_script(array('file'=> array('cp/addons/add-ons-index'),));
```

Comment:
any specific reason for this to be here and not on line 226?

Reply:
Yeah, I had tried putting it on 226 but was not able to get it working there.